.github/licenserc.yaml

```diff
@@ -18,6 +18,7 @@ header:
     - 'kube-monitoring/charts/**' # ignore the license for the kube-monitoring charts
     - 'plutono/charts/**' # ignore the license for the plutono charts
     - 'thanos/charts/templates/tests/**' # ignore the license for this file
+    - 'opentelemetry/chart/templates/tests/**' # ignore the license for the tests of the opentelemetry chart
```

Comment:
I guess we need want to ignore this for all tests? Then let's do `**/templates/tests/**` instead of needing to add each chart individually.

Reply:
The underlying issue has been tracked by @timojohlo here: https://github.com/cloudoperators/greenhouse-extensions/issues/347